lib/logger/browser.js

```diff
@@ -45,8 +45,11 @@ function Logger(options)
     buffer.push(message);
   };
 
+  const doNothingToReconfigure = function (filePath) {
+  };
```

Comment:
I'm not sure what you mean.

This empty function is created only to pass it to line number 52:

```
var common = Core.createLogger(options, logMessage, doNothingToReconfigure);
```

I could replace it by inline function:

```
var common = Core.createLogger(options, logMessage, filePath => {});
```
however I did it as a named function to emphesize the fact that there is nothing to reconfigure for the browser (because we don't create any log file for the browser, we do it only for node logger).

Reply:
I mean that maybe createLogger function could check if third parameter was defined and after that reconfigure only if yes.

Reply:
I changed it.